docs/orleans/streaming/streams-quick-start.md

```diff
@@ -1,27 +1,50 @@
 ---
 title: Orleans streaming quickstart
 description: Learn from the streaming quickstart in .NET Orleans.
-ms.date: 12/06/2022
+ms.date: 01/05/2023
 zone_pivot_groups: orleans-version
 ---
 
 # Orleans streaming quickstart
 
-This guide will show you a quick way to set up and use Orleans Streams.
-To learn more about the details of the streaming features, read other parts of this documentation.
+This guide will show you a quick way to set up and use Orleans Streams. To learn more about the details of the streaming features, read other parts of this documentation.
 
 ## Required configurations
 
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-7-0"
+<!-- markdownlint-enable MD044 -->
+
+In this guide, we'll use a memory-based stream that uses grain messaging to send stream data to subscribers. We will use the in-memory storage provider to store lists of subscriptions, so it is not a wise choice for real production applications.
```

Comment:
I don't really understand why this isn't a wise choice. Is it because it's not persisted in some backend and could be lost? Are you suggesting that this is good for testing but not production? I think the guidance here could be a bit clearer. You may even suggest an alternative that is better with the reason as to why you're using the memory provider.

Reply:
I'll clarify that the memory stream is intended for dem, local development, and test purposes only. Since if the process goes down, none of the streaming content is persisted.